src/Components/DataHistoryExplorer/Internal/TimeSeriesViewer/TimeSeriesViewer.styles.ts

```diff
@@ -0,0 +1,47 @@
+import {
+    ITimeSeriesViewerStyleProps,
+    ITimeSeriesViewerStyles
+} from './TimeSeriesViewer.types';
+
+export const classPrefix = 'cb-time-series-viewer';
+const classNames = {
+    root: `${classPrefix}-root`,
+    commandWrapper: `${classPrefix}-command-wrapper`,
+    command: `${classPrefix}-command`
+};
+export const getStyles = (
+    _props: ITimeSeriesViewerStyleProps
+): ITimeSeriesViewerStyles => {
+    return {
+        root: [
+            classNames.root,
+            { height: '100%', '> div': { height: '100%' } }
```

Comment:
Why did you prefer to use the immediate children selector instead of another className?

Reply:
There is no prop for the wrapper div of the pivot unfortunately, if I don't do that the itemContainer part of the pivot is extending beyond the parent borders:
<img width="696" alt="image" src="https://user-images.githubusercontent.com/45217314/211632074-5a4f3c96-5c02-4ff7-b2d8-1e43bd2567d5.png">